plugin/intero.vim

```diff
@@ -80,4 +84,15 @@ if g:intero_use_neomake
         \ }
 endif
 
+if g:intero_type_on_hover
```

Comment:
Would it make sense to move the conditional inside `type_on_hover()`? One could then enable/disable "type on hover" once in a running session. Now you'd have to re-source or restart Neovim.

Reply:
I guess I was mostly concerned if setting `set updatetime=1000` was something controversial, but there's no problem at all with moving it.

Reply:
Yeah, good point. :)

Reply:
Hmm, how about sort of a middle ground with only `set updatetime=1000` being wrapped in `if g:intero_type_on_hover` and the au command running always (ofc with `intero#repl#type_on_hover()` checking if `g:intero_type_on_hover` is set). 

I'll add some commands to control it all also, so it will reset the updatetime when disabled.